app/scripts/components/analysis/define/use-stac-collection-search.ts

```diff
@@ -89,11 +91,13 @@ export function useStacCollectionSearch({
     );
   }, [datasetLayersInRangeWithNumberOfItems]);
 
-  const unselectableDatasetLayers = useMemo(() => {
+  let unselectableDatasetLayers: DatasetWithTimeseriesData[] = useMemo(() => {
     return datasetLayersInRangeWithNumberOfItems.filter(
       (l) => l.numberOfItems > MAX_QUERY_NUM
     );
   }, [datasetLayersInRangeWithNumberOfItems]);
+  
+  if (invalidDatasets.length) unselectableDatasetLayers = unselectableDatasetLayers.concat((invalidDatasets as unknown) as DatasetWithTimeseriesData[]);
```

Comment:
wouldn't it be more concise to 

```suggestion
  return [...unselectabledatasetLayers, ...invalidDatasets]
```

Reply:
lol yes